here_for_change/templates/base.html

```diff
@@ -25,6 +25,7 @@
     <!-- End Google Tag Manager -->
     {% endif %}
     <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
+    <script src="https://unpkg.com/libphonenumber-js@1.9.9/bundle/libphonenumber-min.js"></script>
```

Comment:
We should monitor the number of 3rd party JS we're loading like this rather than bundling with yarn into the app package. Trade offs either way. For now this is fine.

Reply:
okay, i understand.